pages/linux/ufw.md

```diff
@@ -11,14 +11,26 @@
 
 `ufw disable`
 
-- Add ufw allow rule:
+- Show ufw rules along with their numbers:
 
-`ufw allow {{port}} {{service_name}}`
+`ufw status numbered`
 
-- Add ufw deny rule:
+- Allow incoming traffic on port 5432 on this host:
 
-`ufw deny {{port}} {{service_name}}`
+`ufw allow {{5432}}`
 
-- Show ufw rules:
+- Allow traffic from 192.168.0.4 to any address on this host on port 22 using only tcp protocol:
 
-`ufw status`
+`ufw allow from {{192.168.0.4}} to {{any}} port {{22}} proto {{tcp}}`
+
+- Deny traffic on port 80 on this host:
+
+`ufw deny {{80}}`
+
+- Deny traffic to port 22 using only udp protocol:
+
+`ufw deny from {{any}} to {{any}} port {{22}} proto {{udp}}`
+
+- Remove a particular rule. The rule number can be retrieved from the "ufw status numbered" command:
```

Comment:
That's client-specific (although we *could* declare the node client as *the* spec for clients, we haven't done so, and I don't think we should). In any case, since we're using markdown, it's better to use the markdown syntax to indicate a code snippet.